usr/src/uts/common/brand/lx/syscall/lx_prctl.c

```diff
@@ -278,6 +278,16 @@ lx_prctl(int opt, uintptr_t data)
 		return (0);
 	}
 
+	LX_PR_SET_NO_NEW_PRIVS: {
```

Comment:
Since we're not declaring any new local variables, we don't need the new scope.

Reply:
I was just matching existing style such as with `LX_PR_GET_SECUREBITS`.  Are you cool with leaving it as is?